torch/utils/_python_dispatch.py

```diff
@@ -652,10 +660,16 @@ def get_alias_info(func) -> SchemaInfo:
             )
             for a in func._schema.returns
         ]
-    schema_info = SchemaInfo(args=arg_schemas, outs=out_schemas)
+    schema_info = SchemaInfo(
+        args=arg_schemas, outs=out_schemas, int_tags=[int(x) for x in func.tags]
```

Comment:
would it be reasonable to move this into a `SchemaInfo.__post_init__`, since this is just a perf optimization that can be inferred from the other args? Or are you worried about post_init itself being slow?

Reply:
> are you worried about post_init itself being slow?

Not specifically, more that I had never heard of `__post_init__` before working on this stack.

I would rather execute fewer Python functions than more of them, though, and unless I'm wrong about SchemaInfo being confined to this file I'd rather not compromise performance for this one-callsite class.